examples/todos/pubspec.yaml

```diff
@@ -13,13 +13,13 @@ dependencies:
   flutter_hooks: ">=0.16.0-nullsafety.0 <2.0.0"
   hooks_riverpod:
     path: ../../packages/hooks_riverpod
-  uuid: ^3.0.0-nullsafety.0
+  uuid: ^3.0.4
 
 dev_dependencies:
   build_runner: ^1.9.0
   flutter_test:
     sdk: flutter
-  mockito: ^5.0.0-nullsafety.5
```

Comment:
Why is mockito commented?

Reply:
its not getting use in count example if u want u can re-add it